resources/sample_vaults/Tasks-Demo/.obsidian/snippets/tasks-plugin-query-styling.css

```diff
@@ -0,0 +1,11 @@
+.block-language-tasks:has(ul.plugin-tasks-query-result.tasks-group-by-priority) .tasks-group-heading {
```

Comment:
It's not obvious from the file name that this is part of the manual smoke testing process, and so should not be modified, without changing the contents of the tests as well.

Maybe add a comment to this explaining its purpose?

Reply:
Or ideally, rename it to indicate it is for testing...
I'm generally intending the snippets here to be for users to copy, and they may find this one a bit un-interesting!

Reply:
Actually, I do still think this is valid - most of the existing snippets are useful to users; this one is not.